app/connector/google-sheets/src/main/resources/META-INF/spring.factories

```diff
@@ -0,0 +1,2 @@
+org.springframework.boot.autoconfigure.EnableAutoConfiguration=\
+io.syndesis.connector.sheets.GoogleSheetsVerifierAutoConfiguration
```

Comment:
I see the connector has a verifier but the dependency is not included in the sysndesis meta dependencies list, is that expected ?

Reply:
I thought meta dependency is only needed in case connector uses meta data extension and/or retrieval.

Reply:
yeah, forgot to tell you in our daily meeting :)